agent/integration/job_verification_integration_test.go

```diff
@@ -203,42 +175,51 @@ func TestJobVerification(t *testing.T) {
 	}
 }
 
-func TestWhenTheJobHasASignature_ButTheJobRunnerCantVerify_ItRefusesTheJob(t *testing.T) {
-	t.Parallel()
-
-	job := jobWithValidSignature
+func symmetricJWKFor(t *testing.T, keyMaterial string) jwk.Key {
+	t.Helper()
 
-	// create a mock agent API
-	e := createTestAgentEndpoint()
-	server := e.server(job.ID)
-	defer server.Close()
+	key, err := jwk.FromRaw([]byte(keyMaterial)) // calling FromRaw on a []byte will always yield a symmetric key
+	if err != nil {
+		t.Fatalf("creating jwk: %v", err)
+	}
 
-	mb := mockBootstrap(t)
-	mb.Expect().NotCalled()
-	defer mb.CheckAndClose(t)
+	err = key.Set("alg", "HS256")
+	if err != nil {
+		t.Fatalf("setting alg: %v", err)
+	}
 
-	runJob(t, job, server, agent.AgentConfiguration{}, mb) // note no verification config
+	err = key.Set("kid", keyMaterial) // please don't make the id the key in real life
+	if err != nil {
+		t.Fatalf("setting kid: %v", err)
+	}
 
-	finish := e.finishesFor(t, job.ID)[0]
+	return key
+}
 
-	if got, want := finish.ExitStatus, "-1"; got != want {
-		t.Errorf("job.ExitStatus = %q, want %q", got, want)
+func setOf(t *testing.T, jwkes ...jwk.Key) jwk.Set {
```

Comment:
[more idle bike-shedding]

set of what exactly?

Hungarian notation maybe - `jwksFromKeys(...)`?